graphql/schemabuilder/pagination.go

```diff
@@ -109,7 +114,9 @@ type PaginationArgs struct {
 	After  *string
 	Before *string
 
-	Filter *string
+	Filter    *string
```

Comment:
should we put these in a different embeddable struct? i.e. if you want pagination you embed `PaginationArgs`, `SortArgs`, etc.

Reply:
I'm not sure I like that